homeassistant/components/husqvarna_automower/lawn_mower.py

```diff
@@ -123,22 +98,22 @@ def activity(self) -> LawnMowerActivity:
             return LawnMowerActivity.DOCKED
         return LawnMowerActivity.ERROR
 
-    @handle_sending_exception
+    @handle_sending_exception()
```

Comment:
If you want to see how you can use a decorator with and without brackets, see https://github.com/home-assistant/core/blob/dev/homeassistant/components/http/decorators.py#L47